### ISSUE TITLE
wrapInSafeArea problem

### ISSUE THREAD

**Issue by baltagih2** [FaridSafi/react-native-gifted-chat#2359]
#### Issue Description

On the latest version (2.0.0), the wrapInSafeArea option was removed and it looks like it's now applied automatically. This is causing the input and the keyboard to have a gap between them. I previously had `wrapInSafeArea` set to false which fixed this problem.

#### Steps to Reproduce / Code Snippets

Use the GiftedChat component as usual, if you previously had `wrapInSafeArea={false}`, this doesn't work anymore

#### Expected Results

Being able to set `wrapInSafeArea` to false

#### Additional Information

* Nodejs version: v16.15.0
* React version: 18.2.0
* React Native version: 0.71.3
* react-native-gifted-chat version: ^2.0.0
* Platform(s) (iOS, Android, or both?): tested on iOS simulator and physical device
* TypeScript version: ^4.9.4

**Comment by joecampo** [comment]
@baltagih2 I've run into the same issue. Have you figured out any workaround by chance?

**Comment by joecampo** [comment]
FWIW - I've read through the PR to remove the safe area dependency. I was using wrapInSafeArea={false} anyways.

It creates an issue if you're using tabbed navigation in react-navigation. If the keyboard is open, the tabs are hidden, and a gap is now created between the input and the top of the keyboard. 

Previously I believe this was resolved using the bottomOffset prop, but now it is no longer working. 

I'm still tinkering to see if there are any other ways to fix.

**Comment by baltagih2** [comment]
Any workaround @joecampo in the meantime?

**Comment by joecampo** [comment]
@baltagih2 I'm guessing you're also using React Navigation with tabs and were specifying the `bottomOffset` prop? If so, there isn't a workaround yet, but I submitted a PR to fix: https://github.com/FaridSafi/react-native-gifted-chat/pull/2361

Hopefully we can get it merged and a patch release.

**Comment by baltagih2** [comment]
@joecampo I'm actually not using the `bottomOffset` prop and not using tabs. Regular Stack React Navigation, was using `wrapInSafeArea={false}` and handling everything myself but the prop was removed from the latest release so it's forcing a safeArea around it which breaks my setup

**Comment by joecampo** [comment]
@baltagih2 How big is the gap? If small, even when not using bottomOffset it would set it to `1` when the keyboard was shown even if the prop was not defined. The PR proposed would fix that. (I'm thinking this *might* be your issue).

The safe area dependency was removed. So it is not wrapping in a safe area (just a regular View component) which would be the same behavior as `wrapInSafeArea={false}`. 

If the gap is larger: Is it wrapped in a View component parent? Does the parent have flex: 1? etc.

**Comment by baltagih2** [comment]
![Screenshot 2023-03-21 at 1 38 29 PM](https://user-images.githubusercontent.com/62379853/226595452-ba3af357-1375-4ff9-8c7c-9d725bc743e0.png)

Here is the current gap I'm getting (same one I used to get if I didn't use `wrapInSafeArea={false}`)

**Comment by joecampo** [comment]
@baltagih2 and I'm assuming no gap when the keyboard is closed? Anyways, I'd wait for the PR to be merged and see if that fixes this. I'm not a maintainer of this and have very little experience using this library. 😅

**Comment by baltagih2** [comment]
@joecampo that is correct, no gap with a closed keyboard. Thank you so much for the help, I'll wait for that PR merge and see if it fixes it. Thank you again!

**Comment by baltagih2** [comment]
Update here: In the latest release (2.0.1), I am able to set `bottomOffset` to `22` to remove the gap. However, this is a workaround and I only tested on two devices and it might not work everywhere. Being able to set `wrapInSafeArea` to false would be a cleaner way of doing things in my opinion

**Comment by amerikan** [comment]
@baltagih2 wrapInSafeArea={false} is no longer needed as SafeArea is no longer included in the library in 2.0.0+.

**Comment by amerikan** [comment]
@baltagih2 you can check what changed in 2.0.0 https://github.com/FaridSafi/react-native-gifted-chat/pull/2353/files and 2.0.1 https://github.com/FaridSafi/react-native-gifted-chat/pull/2361/files

**Comment by riehum** [comment]
Does any one fix it or how to tackle it

**Comment by mariomurrent-softwaresolutions** [comment]
Same here, any news about this?

**Comment by felixchan** [comment]
Same here

Huge Gap - latest version @2.6.5

I have SafeAreaView outside, in my parent.

**Comment by kesha-antonov** [comment]
Fixed in new version
